python/cudf_polars/cudf_polars/utils/config.py

```
@@ -69,41 +69,37 @@ def _env_get_int(name: str, default: int) -> int:
 
 
 @functools.cache
-def get_device_handle() -> Any:
+def get_device() -> Any:
     # Gets called for each IR.do_evaluate node, so we'll cache it.
-    import pynvml
+    from cuda.core import system
```

Ruff (0.15.13)

[warning] `import` should be at the top-level of a file

(PLC0415)